.github/scripts/narayana.sh

```diff
@@ -48,29 +48,21 @@ function init_test_variables {
   [ $ARQ_PROF ] || ARQ_PROF=arq	# IPv4 arquillian profile
 
   if [[ $PROFILE == "CORE" ]]; then
-    [ -z "$AS_BUILD" ] && AS_BUILD=1
-    [ -z "$NARAYANA_BUILD" ] && NARAYANA_BUILD=1
-    export AS_BUILD AS_TESTS=0 NARAYANA_BUILD NARAYANA_TESTS=1 XTS_AS_TESTS=0 XTS_TESTS=0 COMPENSATIONS_TESTS=0 txbridge=0
+    export AS_BUILD=1 AS_TESTS=0 NARAYANA_BUILD=1 NARAYANA_TESTS=1 XTS_AS_TESTS=0 XTS_TESTS=0 COMPENSATIONS_TESTS=0 txbridge=0
```

Comment:
I wonder if we were wanting AS_BUILD to be 0 here? https://github.com/jbosstm/narayana/pull/2453/files#diff-a86bb2175b62f05d86a7f7fe0f3fd6d6c44e2cdfd0f4e4a92759c817d7959d96L101

Reply:
I am going to give an approval because I see JTA_AS_TESTS=1 on the line below - but please do have a look

Reply:
AFAIU, `AS_BUILD` will be zeroed by `init_test_variables` at line [92](https://github.com/jbosstm/narayana/pull/2453/files#diff-4d1f4fcd4e58349f054d9e7e7887b4f21e0684fe0b192e85a6e291e34fcef2ceL92). We don't use the `PROFILE` env. variable in our GHA workflows; instead, we set the specific env. variables that activate specific tests (note: we use `PROFILE` only for those axes where we don't build WildFly IIRC)